jsapp/js/projects/projectsTable/projectsTableHeader.module.scss

```diff
@@ -55,3 +61,92 @@
     }
   }
 }
+
+// Resizers
+
+// Mechanics
+$resizer-width: sizes.$x16; // Keep this at least 10px for ease of use
+// Appearance
+$resizer-line-width: sizes.$x1;
+$resizer-line-vertical-offset: sizes.$x4;
+$resizer-area-border-radius: sizes.$x4;
+$cell-line-width: sizes.$x1;
+$cell-line-vertical-offset: sizes.$x6;
+// Theme colors
+$cell-line-color: colors.$kobo-gray-85;
+$resizer-line-color: colors.$kobo-blue;
+$resizer-area-color: colors.$kobo-light-blue;
+
+// Resizer "pad" - interactive area
+.resizer {
+  cursor: ew-resize; // col-resize <-|->   ew-resize <->
```

Comment:
I asked Ignacio which cursor he preferred, and he liked the `ew-resize` version for hover.

When I added the full-body cursor during drag, I switched it to `col-resize`, and I'm pleased with the way the two work in concert.